### PR TITLE
cases: include multi-machine scenario in Caching Hub

### PR DESCRIPTION
**Pull request by jorgeorpinel**
per https://github.com/iterative/dvc.org/pull/2791#pullrequestreview-764848551

> You may disregard these recommendations if you used the **Edit on GitHub** button from dvc.org to improve a doc in place.

❗ Please read the guidelines in the [Contributing to the Documentation](https://dvc.org/doc/user-guide/contributing/docs) list if you make any substantial changes to the documentation or JS engine.

🐛 Please make sure to mention `Fix #issue` (if applicable) in the description of the PR. This causes GitHub to close it automatically when the PR is merged.

Please choose to [allow us](https://help.github.com/en/github/collaborating-with-issues-and-pull-requests/allowing-changes-to-a-pull-request-branch-created-from-a-fork) to edit your branch when creating the PR.

Thank you for the contribution - we'll try to review it as soon as possible. 🙏

### CODE REVIEW

File: content/docs/use-cases/fast-data-caching-hub.md [hunk 1]
```diff
@@ -12,8 +12,8 @@ DVC's built-in data <abbr>caching</abbr> lets you implement a simple and
 efficient storage layer globally -- for your entire team. This approach can help
 to
 
-- speed up data transfers from massive object stores currently on the cloud, or
-  move a growing dataset out of your machine without slowing things down.
+- speed up data transfers from massive object stores on the cloud, or share data
+  across multiple on-premise machines without slowing things down.
```
Comment (shcheklein) [inline content/docs/use-cases/fast-data-caching-hub.md:16]: `on-prem` - is not the right term, just machines is enough I think

**Comment by shcheklein** [comment]
We might need to adjust the image?

File: content/docs/use-cases/fast-data-caching-hub.md (hunk 1 above)
Comment (jorgeorpinel) [reply content/docs/use-cases/fast-data-caching-hub.md:16]: OK. I thought we wanted to emphasize a local/virtual network team environment but yeah on-prem is not the best term for that either. Removing for now

```suggestion
  across multiple machines without slowing things down.
```

Comment (jorgeorpinel) [reply content/docs/use-cases/fast-data-caching-hub.md]: Or maybe "multiple team member environments"

Comment (jorgeorpinel) [reply content/docs/use-cases/fast-data-caching-hub.md]: ?

Comment (shcheklein) [reply content/docs/use-cases/fast-data-caching-hub.md]: multiple machines sounds good to me :)

**Review by shcheklein — APPROVED** [review]

**Comment by jorgeorpinel** [comment]
> We might need to adjust the image?

This one?
image

Right. It only shows multiple users right now but it could be more explicit and show 2 or 3 environments or machines but on the other hand that's what we had previously and wanted to simplify it.

**Pull request merged**